service/worker/scheduler/fx.go

```diff
@@ -24,6 +24,9 @@ import (
 const (
 	WorkflowType      = "temporal-sys-scheduler-workflow"
 	NamespaceDivision = "TemporalScheduler"
+
+	// ChasmSchedulerExperiment is the experiment name for enabling CHASM (V2) scheduler
+	ChasmSchedulerExperiment = "chasm-scheduler"
```

Comment:
I don't think this is a good place for the constant. I would probably create a dedicated file to list all of the experiments or go the other direction and put it in the CHASM library.